docs/_includes/variation-content.html

```diff
@@ -45,7 +45,9 @@ <h2>
             <a class="m-list_link" href="#variations">Variations</a>
         </li>
         {% endif %}
-        {% if page.usage %}
+        <!-- liquid templates' order of operations is from right to left, see https://help.shopify.com/en/themes/liquid/basics/operators for examples explaining the nonsense below -->
```

Comment:
I know it's fugly but I think it makes sense to use [liquid comments](https://github.com/cfpb/design-system/blob/983d8a6087796a778b9b0e9abef8d051acc3c466/src/docs/_includes/section-nav.html#L6) when referencing liquid code and html comments for html.